notes/lecture14.tex

```diff
@@ -0,0 +1,273 @@
+\section{Lecture 14: Algorithms using Duality}
+
+The Lagrangian duality theory from the previous lecture can be used to
+design improved optimization algorithms which perform the optimization
+on the dual function. Oftentimes, passing to the dual can simplify
+computation or enable parallelization.
+
+\subsection{Review}
+
+Recall the \emph{primal problem}
+\[
+\min_{x \in \Omega} f(x) : A x = b
+\]
+The corresponding dual problem is obtained by considering the
+\emph{Lagrangian}
+\[
+L(x,\lambda) = f(x) + \lambda^T(A x  - b)
+\]
+where $\lambda_i$ are called \emph{Lagrange multipliers}. The
+\emph{dual function} is defined as
+\[
+g(\lambda) = \inf_{x \in \Omega} L(x,\lambda)
+\]
+and the \emph{dual problem} is
+\[
+\sup_{\lambda \in \R^m} g(\lambda)
+\]
+
+\begin{definition}[Concave functions]
+$f$ is concave $\iff -f$ is convex.
```

Comment:
Don't start sentence with formula. Write instead "A function $f$ is..."

Reply:
:+1: